src/view.js

```diff
@@ -96,7 +96,7 @@ const View = Backbone.View.extend({
     Backbone.View.prototype.setElement.apply(this, arguments);
 
     if (hasEl) {
-      this._isRendered = !!this.$el.length;
+      this._isRendered = this.Dom.hasContents(this.el);
```

Comment:
Why not `this._isRendered = this.el.hasChildNodes()`?

I dont see a different implementation warranting an abstraction and is much simpler